forest/shared/src/cli/mod.rs

```diff
@@ -103,6 +103,12 @@ pub struct CliOpts {
     /// Turn on tokio-console support for debugging
     #[structopt(long)]
     pub tokio_console: bool,
+    /// Send telemetry to `grafana loki`
+    #[structopt(long)]
+    pub loki: bool,
+    /// Endpoint of `grafana loki`
+    #[structopt(long, default_value = "http://127.0.0.1:3100")]
```

Comment:
It would be nice to use the same convention like for metrics and rpc endpoints, i.e. `Option<SocketAddr>`, just for consistency.

Reply:
This one is a bit different, it's served externally so the protocol (http/https) has to be specified, while the other endpoints are served by forest daemon and http is implicitly specified